pulp_ansible/app/viewsets.py

```diff
@@ -245,6 +245,11 @@ class CollectionRemoteViewSet(RemoteViewSet):
     queryset = CollectionRemote.objects.all()
     serializer_class = CollectionRemoteSerializer
 
+    def update(self, request, pk, **kwargs):
+        if "requirements_file" in request.data:
+            request.data["last_synced_metadata_time"] = None
```

Comment:
if we change the `requirements_file` unset `last_synced_metadata_time`.

for avoiding no-op when a new requirements file contains more collections than the previous one

Reply:
I think this is good. I didn't think about this. :+1:

Reply:
Can a test be added for this part please?